ajax/libs/thrift/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "thrift",
+  "filename": "thrift.min.js",
+  "description": "The Apache Thrift software framework, for scalable cross-language services development, combines a software stack with a code generation engine to build services that work efficiently and seamlessly between C++, Java, Python, PHP, Ruby, Erlang, Perl, Haskell, C#, Cocoa, JavaScript, Node.js, Smalltalk, OCaml and Delphi and other languages.",
+  "homepage": "https://thrift.apache.org/",
+  "keywords": [
+    "thrift",
+    "rpc",
+    "framework",
+    "serialization",
+    "cross-language"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://git-wip-us.apache.org/repos/asf/thrift.git"
+  },
+  "version": "0.10.0",
+  "author": {
+    "name": "Apache Thrift Developers",
+    "email": "dev@thrift.apache.org",
+    "url": "https://thrift.apache.org"
+  },
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://git-wip-us.apache.org/repos/asf/thrift.git",
```

Comment:
I think for auto-update mechanism both are acceptable. I run auto-update and both succeed.